downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -65,7 +65,7 @@ class Git : GitBase() {
             val gitInfoDir = File(targetDir, ".git/info").apply { safeMkdirs() }
             val path = vcs.path.let { if (it.startsWith("/")) it else "/$it" }
             File(gitInfoDir, "sparse-checkout").writeText("$path\n" +
-                    LicenseFileMatcher.DEFAULT_NAMES.joinToString("\n") { "/$it" })
+                    LicenseFileMatcher.DEFAULT_MATCHER.fileNames.joinToString("\n") { "/$it" })
```

Comment:
This commit effectively reverts a48cc030a7dfe877ac0c6c1185dd8cbc3aee3985. I consider the current code to be cleaner as the code locations where the default file names are used requires less code.

Also, I do not really get the "inconsistencies" argument from the commit message. Could you give a concrete example where having both `DEFAULT_NAMES` and `DEFAULT_MATCHER` could lead to inconsistencies? Both are immutable properties so they should always be in sync.

Reply:
I have squashed the commits. I'm not sure if you still consider it to be a problem after the squash - thus not addressing it for now.